extensions/amp-story/1.0/audio.js

```diff
@@ -33,10 +33,7 @@ export function upgradeBackgroundAudio(element, loop = true) {
     return null;
   }
   const audioEl = element.ownerDocument.createElement('audio');
-  const audioSrc = Services.urlForDoc(element).assertHttpsUrl(
-    element.getAttribute('background-audio'),
-    element
-  );
+  const audioSrc = element.getAttribute('background-audio');
```

Comment:
Hi @apatronl! Thanks for the contribution :) 

We do this check to be sure the audio file is being served securely through https. 
It looks like this commit would remove the check.

Local audio files can be tested while developing. The `assertHttpsUrl` provides an exception for localhost:
https://github.com/ampproject/amphtml/blob/main/src/service/url-impl.js#L120-L133
 
Please let me know if you have any other questions about the check and why it's necessary :)